### DISCUSSION TITLE
Remove Master Authentication Password

### DISCUSSION THREAD

**Discussion by AK-survey** [opengisch/QField#3920] [Q&A]
I have been using QField and Qfield Sync for a while, but recently tried uploading a project using QFieldCloud. At some point, the software asked me to create a master authentication password, which I did, not knowing what it was really about. My issue here is I don't think I need this and it asks for the password every time I open QGIS. I have uninstalled the plugin and restarted the computer, and I still get this same pop-up window as long as QField Sync plugin is enabled.

![image](https://user-images.githubusercontent.com/113943591/219483932-56ffe33d-f66a-466f-b314-984a064a21a9.png)

I can click cancel, but it's very annoying that it even pops up. I know it's related to QFieldSync plugin because if I disable the plugin, I don't get the pop-up, but as soon as I re-install the plugin, there's the pop-up again! 

Please inform me of the purpose of this password (if it is required for something), and more importantly, how to remove it so I don't get the pop-up every time I open QGIS. I'm running QGIS v3.26.3 and QFieldSync v4.3.1.

Thanks in advance,
David

**Comment by ZsanettMed** [comment]
Hey David, 
Just store the password in your Password Manager! When you enter the password, there's a checkbox to that effect.

<img width="285" alt="image" src="https://user-images.githubusercontent.com/83091224/219613191-b6c99e26-26f9-41d8-91f8-739a85a10221.png">

Best, 
Zsanett

**Reply by AK-survey** [reply]
Zsanett,

That would be all fine and good except 2 things. First off, I don't use a password manager. Second, it doesn't change the fact that I don't know why this is popping up and what it's even really for. I'd like to disable whatever it is that's requiring it in the first place.

-David

**Reply by BoswachterMarc** [reply]
I think it is QGis related... https://docs.qgis.org/3.22/en/docs/user_manual/auth_system/auth_overview.html?highlight=authentication
